lms/djangoapps/grades/new/course_grade.py

```diff
@@ -112,6 +166,10 @@ def summary(self):
         # doesn't get displayed differently than it gets grades
         grade_summary['percent'] = self.percent
         grade_summary['grade'] = self.letter_grade
+
+        # These two fields are stored separately from percent and grade. The should match unless accessed in between
+        # course and subsection grade updates. If that happens we re-populate them here.
+        # TODO: re-run tests that hit this code after Eric's PR lands without explicitly populating grades
```

Comment:
I don't think this applies anymore now that chapter_grades is calculated lazily.